uiowa/uiowa
# Feature: person_extended config split updates

* Updated website field to allow multiple entries.
* Updated the title field of the link to display as 'Label' in forms.
* Added descriptive help text to guide user with adding multiple websites.

# To test
* `blt ds --site=informatics.grad.uiowa.edu`
* `drush @gradinformatics.local uli`
* https://gradinformatics.local.drupal.uiowa.edu/node/add/person
* Observe the ability to add more than one website
* Observe help text
* Observe link title field label is "Label"
